ci/ansible.cfg

```diff
@@ -0,0 +1,5 @@
+[defaults]
+callback_whitelist = custom_logger
+# additional paths to search for roles 
+roles_path = ~/feature-verification-tests/roles:./roles:/usr/share/ansible/roles:/etc/ansible/roles:~/.ansible/roles:/home/zuul/src/github.com/openstack-k8s-operators/ci-framework/roles
```

Comment:
```suggestion
roles_path = /home/zuul/src/github.com/infrawatch/feature-verification-tests/roles:/usr/share/ansible/roles:/etc/ansible/roles:~/.ansible/roles
```

Reply:
I needed openstack-k8s-operators/ci-framework/roles for running  the task cluster login from the role : rhol_crc

Reply:
Of course. I saw that and didn't join the dots.